src/components/pokemon/Pokemon.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import './Pokemon.css';
+
+function Pokemon({ endpoint }) {
+  const [pokemon, setPokemon] = useState({});
+
+  useEffect(() => {
+    console.log(endpoint);
+
+    async function fetchData() {
+      try {
+        const { data } = await axios.get(endpoint);
+        setPokemon(data);
+      } catch (e) {
+        console.error(e);
+      }
+    }
+
+    if (endpoint) {
+      fetchData();
+    }
+  }, [endpoint]);
+
+
+  return (
+    <section className="poke-card">
+      {Object.keys(pokemon).length > 0 &&
```

Comment:
Dan kan deze check ook iets simpeler
```suggestion
      {pokemon &&
```